src/Containers/Impl/ContainerServiceNotRunningException.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.R.Containers {
```

Comment:
Maybe just have a `ContainerError ` enum and the `Error` field in the `ContainerException` ?